adding beta t&c (#3782)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/experimental-features.md

2. Find Experimental features at the bottom of Your Profile page.
3. Click **Beta** to toggle the features on or off as shown in the following image.
   ![Experimental features](/img/docs/dbt-versions/experimental-feats.png)

## Beta terms and conditions

> By using or enabling features that aren't yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).

Author: not sure I need the 'yet' here. Maybe '..arent available in general release'??

> By using or enabling features that aren't yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).

Author: ```suggestion
By using or enabling features that are not yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).
```

> By using or enabling features that are not yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).

Reviewer: ```suggestion
By using or enabling features that are not yet in general release (beta features), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).
```

Author: nice one, thanks @nghi-ly ! i do agree with you here but I'm going to stick with "Beta Features" because that's how it's presented [legally](https://deploy-preview-3782--docs-getdbt-com.netlify.app/assets/files/beta-tc-740ff696113c89c38a96bb70b968775e.pdf) in the pdf so I'd like to be consistent there. 

I've accepted your other suggestion though so let me know if you have any strong feelings on that!

Reviewer: @mirnawong1 : sounds good to me! 👍

> By using or enabling features that aren't yet in general release ("Beta Features"), you agree to its terms and conditions. Download the [Beta terms and conditions](/assets/beta-tc.pdf) for more info.  

Author: ```suggestion
By using or enabling features that aren't yet in general release ("Beta Features"), you agree to the [Beta Features Terms and Conditions](/assets/beta-tc.pdf).
```
